entwatch/ze_ffxii_mt_bur_omisace_v6.jsonc

```diff
@@ -0,0 +1,511 @@
+        "shortname": "Bow",
+        "hammerid": "5066",
+        "message": true,
+        "ui": true,
+        "transfer": true,
+        "color": "grey",
+        "handlers": [
+            {
+                "type": "button",
+                "hammerid": "5067",
+                "event": "OnPressed",
+                "mode": 2,
+                "cooldown": 80,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },
+    {
+        "name": "Ultimate Blade",
+        "shortname": "Blade",
+        "hammerid": "6436",
+        "message": true,
+        "ui": true,
+        "transfer": true,
+        "color": "yellow",
+        "handlers": [
+            {
+                "type": "button",
+                "hammerid": "6171",
+                "event": "OnPressed",
+                "mode": 3,
+                "cooldown": 0,
+                "maxuses": 1,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },
+    {
+        "name": "Zombie Judge Drace",
+        "shortname": "ZM Drace",
+        "hammerid": "3275",
+        "message": true,
+        "ui": true,
+        "transfer": false,
+        "color": "red",
+        "triggers": ["3276"],
+        "handlers": [
+            {
+                "type": "game_ui",
+                "hammerid": "3271",
+                "event": "OnTrigger",
+                "mode": 2,
+                "cooldown": 70,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "game_ui",
+                "hammerid": "3270",
+                "event": "OnTrigger",
+                "mode": 2,
+                "cooldown": 65,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },
+    {
+        "name": "Zombie Judge Gabranth",
+        "shortname": "ZM Gabranth",
+        "hammerid": "3886",
+        "message": true,
+        "ui": true,
+        "transfer": false,
+        "color": "red",
+        "triggers": ["3876"],
+        "handlers": [
+            {
+                "type": "game_ui",
+                "hammerid": "3882",
+                "event": "OnTrigger",
+                "mode": 2,
+                "cooldown": 65,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            },
+            {
+                "type": "game_ui",
+                "hammerid": "3881",
+                "event": "OnTrigger",
+                "mode": 2,
+                "cooldown": 65,
+                "maxuses": 0,
+                "message": true,
+                "ui": true
+            }
+        ]
+    },
+    {
+        "name": "Zombie Damage Magick",
+        "shortname": "ZM Damage",
+        "hammerid": "3102",
+        "message": true,
+        "ui": true,
+        "transfer": true,
```

Comment:
I can't confirm right now because the workshop submission got temporarily hidden, but pretty sure a ZM item is going to be a knife and non-transferable.